openmetadata-ui/src/main/resources/ui/src/utils/IngestionUtils.tsx

```diff
@@ -191,7 +192,10 @@ export const getIngestionTypes = (
   ];
 };
 
-const getPipelineExtraInfo = (pipelineType?: PipelineType) => {
+const getPipelineExtraInfo = (
+  isPlatFormDisabled: boolean,
+  pipelineType?: PipelineType
+) => {
```

Comment:
as you are changing the position of the `pipelineType`, please check all the places where it is used